wiki/en/en-Getting-Started.md

```diff
@@ -11,21 +11,21 @@ permalink: "/wiki/Getting-Started"
 
 1. **A computer** running either **Windows 10**, **macOS** (**MacOS X** El Capitan v10.11 or higher), or **Linux**.
 1. **A wired internet connection** (don't use Wi-Fi, it will cause sound problems). Most broadband connections will be fine[^1].
-1. **Wired headphones - not speakers** to prevent echo or feedback and to minimize delay[^2].
+1. **Wired headphones - not speakers** to prevent echo or feedback and to minimise delay[^2].
```

Comment:
I think we already had a discussion about BE/AE and I think minimize is also ok in BE?

Reply:
Minimise is preferred in BE.